tests/integration/api/server.js

```diff
@@ -14,16 +15,14 @@ describe('server', () => {
       };
 
       return utils.requestOnTestDb(opts, true)
-        .then(fail)
+        .then(() => assert.fail('should have thrown'))
```

Comment:
is `assert` a global here? Can we include it in the require, so we're 100% sure which `assert` we're using?